thirdeye/thirdeye-frontend/app/pods/components/metrics-table/component.js

```diff
@@ -13,40 +13,46 @@ export default Component.extend({
     }, {
       propertyName: 'label',
       title: 'Metric',
-      className: 'rootcause-metric__table__column'
+      className: 'rootcause-metric__table__column rootcause-metric__table__links-column--large'
```

Comment:
So many column configs... so much writing. It almost sounds like you could use a util function to build out these columns! <Insert shameless self-plug here> :D
In all seriousness, not a big deal but if you end up having multiple tables and want to have more readable configs, callisto and cmon-web both use artdeco table config builders